order-dev.js

```diff
@@ -0,0 +1,488 @@
+    $('div.customerReply').replaceWith(`
+      <div class="customerReply">
+        <div class="section">
+          <div class="replyTitle"><h5>Reply:</h5></div>
+          <div class="spinner"></div>
+          <form id="reply" id="replyForm">
+            <textarea class="replyText" form="replyform" placeholder="Sending reply...\n${replyText}"></textarea>
+          </form>
+        </div>
+      </div>
+    `);
+    console.log("\nCALL v2-custReply: " + new Date());
+    API.call({
+      cacheMS: -1,
+      method: 'v2-customerReplyToFreshdesk',
+      httpMethod: 'POST',
+      body: JSON.stringify({replyText: reply}),
+      onSuccess: (data) => {
+        console.log("\nv2-custReply SUCCESS: " + new Date());
+        console.log("\nDATA stringified = " + JSON.stringify(data));
+
+        if(data.solveRes.records[0].ticket_id) {
+          location.reload();
+        }else{
+          console.log("\nNO solve response");
+        }
+      },
+      onFailure: (data) => {
+        console.log('customer reply fail');
+        $('#main').html(`
+          <div class="row">
+            <p>An error has occured.<br>${data.error}<br>Please try again later.</p>
+          </div>
+        `);
+      }
+    });
+  }else{//no reply text when submit button hit. try again
+    $('div.customerReply').replaceWith(`
+        <div class="customerReply">
+          <div class="section">
+            <div class="replyTitle"><h5>Reply:</h5></div>
+            <form id="reply" id="replyForm">
+              <textarea class="replyText" form="replyform" placeholder="No reply found. Enter Reply here:\n${replyText}"></textarea>
+            </form>
+            <button class="replyButton" onclick="createReplyTicket()">Submit</button>
+          </div>
+        </div>
+      `);
+  }
+}
+
+function solveTimeString(date) {
+  var hour = date.getHours();
+  var ampm = (hour > 11) ? `pm` : `am`;
+  hour = `${date.getHours() === 0 || date.getHours() === 12 ? '12' : (date.getHours() + 24) % 12}`;
+  var min = `${date.getMinutes() < 10 ? '0' : ''}${date.getMinutes()}`;
+  return result = `${hour}:${min}${ampm}`;
+}
+
+var autoExpand = function (field) {//function to auto grow textarea
+
+	field.style.height = 'inherit';// Reset field height
+
+	var computed = window.getComputedStyle(field);// Get the computed styles for the element
+
+	var height = parseInt(computed.getPropertyValue('border-top-width'), 10)// Calculate the height
+	             + parseInt(computed.getPropertyValue('padding-top'), 10)
+	             + field.scrollHeight
+	             + parseInt(computed.getPropertyValue('padding-bottom'), 10)
+	             + parseInt(computed.getPropertyValue('border-bottom-width'), 10);
+
+	field.style.height = height + 'px';
+};
+
+document.addEventListener('input', function (event) {
+	if (event.target.tagName.toLowerCase() !== 'textarea') return;
+	autoExpand(event.target);
+}, false);
+
+function convertReply(reply) {
+	var input_str = reply.trim(); //store input
```

Comment:
whoa indents...

Reply:
oh, if this is someone else's function, should be kept whole so that is clear. then you invoke that function. Like the autoExpand one.